README.md

```diff
@@ -129,6 +134,14 @@ Must supply only one of the following
     - `endpoints` is the list of etcd nodes in your cluster
     - `prefix` is the path that will be prefixed to all service discovery keys. This key is optional. (Default: `/containerbuddy`)
 
+Logging Config:
+
+The logging config adjust the output format and verbosity of Containerbuddy logs.
+
+- `level` adjusts the verbosity of the messages output by containerbuddy. Must be one of: DEBUG, INFO, WARN, ERROR, FATAL, PANIC (Default: INFO)
+- `format` adjust the output format for log messages. Can be "go", "text", or "json" (Default: "go")
```

Comment:
What do you think about including a link to the descriptions for these formats? These names are logrus-specific and "Go developers who use logrus" aren't the only audience for Containerbuddy.